configs/recognition/tsn/README.md

```diff
@@ -26,6 +26,21 @@
 
 Here, We use [1: 1] to indicate that we combine rgb and flow score with coefficients 1: 1 to get the two-stream prediction (without applying softmax).
 
+### Kinetics-400 Data Benchmark (8-gpus, ResNet50, ImageNet pretrain; 3 segments)
+
```

Comment:
pls add a short description, explaining what "data benchmark" means. and explain which variations are compared?